src/routes/boot.ts

```diff
@@ -130,6 +136,13 @@ type BootMiddleware = (
   res: FastifyReply,
 ) => Promise<Record<string, unknown>>;
 
+const geoSection = (req: FastifyRequest): BaseBoot['geo'] => {
+  return {
+    region: req.headers['x-client-region'] as string,
+    ip: req.ip,
```

Comment:
Do we need the ip to be returned publicly? Even with boot being a private endpoint it opens the attack vector which can dox the client's IP.

Reply:
Originally, I planned to provide the IP to paddle for the localization but decided to go with a more privacy-oriented way once I found this geolocation trick. We can remove it and there will be no breaking changes

Reply:
https://github.com/dailydotdev/daily-api/pull/2465